src/PhysicsModel.cpp

```diff
@@ -169,6 +169,12 @@ bool PhysicsModel::loadMatrixElements(
 {
     mMatrixElements.clear();
 
+    if (mOffEqIndices.size() < 1)
+    {
+        std::cout << "Warning: PhysicsModel contains no out-of-equilibrium particles, loadMatrixElement() does nothing (treated as success)." << std::endl;
```

Comment:
Should this be a success and what values for v_w would I expect then? Shouldn't it be the LTE result if the Collisiontensor is empty? Perhaps false return is better here? However, it would never be very costly since it doesn't require integration.

Reply:
That is separate concern from here. Calling createCollisionTensor() on a PhysicsModel that has no valid matrix elements loaded prior to it will produce an empty CollisionTensor so its not very useful. I kinda like the idea of allowing empty objects to still be valid so opted for a "success" loading of matrix elements if the model is empty. But currently empty objects will error out somewhere later I think.

If the collision term in Boltzmann eq is 0 I think the result is opposite of LTE, since there's no thermalization without collisions.